homeassistant/components/point/__init__.py

```diff
@@ -206,12 +263,17 @@ async def _sync(self):
         async def new_device(device_id, platform):
             """Load new device."""
             config_entries_key = f"{platform}.{DOMAIN}"
-            async with self._hass.data[DATA_CONFIG_ENTRY_LOCK]:
-                if config_entries_key not in self._hass.data[CONFIG_ENTRY_IS_SETUP]:
-                    await self._hass.config_entries.async_forward_entry_setups(
-                        self._config_entry, [platform]
+            async with self._config_entry.runtime_data[DATA_CONFIG_ENTRY_LOCK]:
+                if (
+                    config_entries_key
```

Comment:
What does the config_entries_key resemble?

Reply:
I guess that is the auth-key.

Reply:
The `config_entries_key` is a unique identifier for the current device. But it looks stange..

Reply:
I can take a look at this when we get the oauth code merged